frontend/src/components/models.ts

```diff
@@ -71,7 +71,7 @@ export const supportedChains: Array<Chain> = [
     logoURI: ETH_NETWORK_LOGO,
```

Comment:
*Arbitrary comment location*

Given that the toll is 0.05 xDAI, I think the minimum send amount should probably be set to something like 0.5 xDAI


<img width="535" alt="Screen Shot 2023-06-13 at 3 25 27 PM" src="https://github.com/ScopeLift/umbra-protocol/assets/1479802/5d5f4a80-5eb8-4d93-8cb3-d9aa593dae99">